deployment/docker-compose.yml

```diff
@@ -74,6 +74,7 @@ services:
         -Dorg.slf4j.simpleLogger.logFile=System.err
       UNTRUE_PLAN_START: "2000-01-01T11:58:55.816Z"
     image: "${REPOSITORY_DOCKER_URL}/aerie-merlin-worker:${DOCKER_TAG}"
+    ports: ["27187:8080"]
```

Comment:
~~This ports line seems spurious~~

Nevermind, good catch! As a reviewer, this caught my eye as an "unrelated change" - I won't request it be split out into its own commit now, but just a note for the future.